project/Dependencies.scala

```diff
@@ -107,9 +107,9 @@ object Dependencies {
   val log4jSlf4jImpl = log4jModule("log4j-slf4j-impl")
   val log4jModules = Vector(log4jApi, log4jCore, log4jSlf4jImpl)
 
-  val caffeine = "com.github.ben-manes.caffeine" % "caffeine" % "2.8.5"
+  val caffeine = "com.github.ben-manes.caffeine" % "caffeine" % "3.2.1"
```

Comment:
Not sure why CI didn't catch this, but we can't use caffeine 3.x because we're still on JDK 8.

Reply:
I guess because we do not run CI on JDK 8 - https://github.com/sbt/sbt/pull/8169